packages/app/components/chat/index.tsx

```diff
@@ -14,12 +14,14 @@ import {
   RStack,
   SubmitButton,
 } from '@packrat/ui';
-// import { sendMessage } from '@packrat/validations';
+import { sendMessage } from '@packrat/validations';
 import useCustomStyles from 'app/hooks/useCustomStyles';
 import { useChat } from 'app/hooks/chat/useChat';
 import { loadStyles } from './chat.style';
 // import { Select } from "tamagui";
 
+// TODO check if we fixed chat screen on another branch
```

Comment:
@taronaleksanian I suggest you create an issue on the github repository and paste the link of this created issue here.

Reasons: 
- It helps by providing more context about the task
- We can easily keep track of the tasks that need to be done.

```suggestion
// TODO check if we've fixed the chat screen on another branch
// link: https://github.com/andrew-bierman/PackRat/issues/ ???
```